examples/webhook-signing/nestjs/main.ts

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env -S npm run-script run
+
+import { NestFactory } from "@nestjs/core";
+import { INestApplication } from "@nestjs/common";
+import { AppModule } from "./app.module";
+
+async function bootstrap() {
+  const app = await NestFactory.create<INestApplication>(AppModule, { rawBody: true });
+  app.enableCors({
+    origin: "*",
+  });
+
+  await app.listen(3000, () => {
```

Comment:
Is it possible to bind to port 0 and print out the dynamically assigned port? This helps with port conflicts when multiple tests are running in parallel. See https://github.com/stripe/stripe-node/blob/master/examples/webhook-signing/express/main.ts#L71-L76 for example.

Reply:
It is now assigned to a dynamic port